packages/devtools_app/test/evaluate_test.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/debugger/debugger_model.dart';
+import 'package:devtools_app/src/debugger/evaluate.dart';
+import 'package:devtools_app/src/ui/search.dart';
+import 'package:flutter/widgets.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+import 'package:vm_service/vm_service.dart';
+
+import 'support/mocks.dart';
+
+void main() {
+  MockDebuggerController debuggerController;
+
+  setUp(() async {
+    debuggerController = MockDebuggerController();
+
+    when(debuggerController.selectedStackFrame).thenReturn(
+      ValueNotifier(
+        StackFrameAndSourcePosition(
+          Frame(
+            index: 0,
+            location: SourceLocation(
+                script: ScriptRef(id: 'some-script-loc', uri: ''), tokenPos: 0),
+          ),
+        ),
+      ),
+    );
+    when(debuggerController.variables).thenReturn(
+      ValueNotifier(
+        [
+          Variable.create(
+            BoundVariable(
+                name: 'foo',
+                value: null,
+                declarationTokenPos: 0,
+                scopeStartTokenPos: 0,
+                scopeEndTokenPos: 0),
```

Comment:
trailing comma for all these BoundVariable, FieldRef, Class, FuncRef, etc. instances